### DISCUSSION TITLE
Try to upload to supabase storage but throwing error 401

### DISCUSSION THREAD

**Discussion by git-create-devben** [supabase/supabase#18714] [General]
This is my first time using supabase So I tried to upload to a storage but seeing this error in my console and I really don't have any idea what it's already search online but couldn't fix it I'll really appreciate any of your opinions.

 statusCode: '401', error: 'Invalid JWT', message: 'new row violates row-level security policy for table "objects"'

**Comment by GaryAustin1** [comment]
This means you have not set your storage policies to allow insert OR you are not meeting the policy.  All storage buckets need RLS policies to do operations on EXCEPT for using a publicURL to access a file in a public bucket.

**Reply by GaryAustin1** [reply]
Did you get this working?

**Reply by git-create-devben** [reply]
No, but have look for an alternative. Thanks man
